lib/fetchers/registry/manifest.js

```diff
@@ -75,5 +75,9 @@ function annotateManifest (spec, manifest, opts) {
     manifest._resolved =
     `${registry}/${manifest.name}/-/${manifest.name}-${manifest.version}.tgz`
   }
+  // NOTE - Workaround for https://npm.community/t/285
+  if (manifest._resolved && manifest._resolved.startsWith('http://registry.npmjs.org/')) {
+    manifest._resolved = manifest._resolved.replace('http', 'https')
```

Comment:
You may want to use `replace('http://', 'https://')` so that if you have a package with `http` in the name it will not replace it

Reply:
This should only have replaced the first occurrence, and since we've tested it to start with `http://registry.npmjs.org/` it should be okay. However, that's presuming `replace` with a string parameter isn't automatically global.

Either way, the underlying problem is fixed now.

Reply:
You are correct - did not realize to "replace all" you need the regex global flag